Intermittent build error

### Current behavior

Created new project using the latest dev template (eg `dotnet new unoapp`)
Launched solution in VS
Waited for solution to load and then clicked reload button in notification at top of editor area
Rebuilt Mobile project
Unloaded and reloaded Mobile project 
Set Mobile project as Startup project (and Android emulator is set as deploy target)
Rebuilt the Mobile project
Build error similar to the following

2>C:\Users\XXXX\.nuget\packages\uno.winui\5.0.0-dev.3568\buildTransitive\Uno.UI.Tasks.targets(83,3): warning UNOB0003: Ignoring resource **\*.resw, could not determine the language
2>C:\Users\XXX\.nuget\packages\uno.winui\5.0.0-dev.3568\buildTransitive\Uno.UI.SourceGenerators.props(366,66): error MSB4184: The expression "[System.IO.Path]::GetFullPath(**\*.resw)" cannot be evaluated. Illegal characters in path.


### Expected behavior

No build errors
[AsdAsd2_Debug_AnyCPU_Build_2023-10-20T18_28_23.8085604+11_00.zip](https://github.com/unoplatform/uno/files/13051437/AsdAsd2_Debug_AnyCPU_Build_2023-10-20T18_28_23.8085604%2B11_00.zip)


### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

--- a/src/Uno.Templates/Uno.Templates.csproj
+++ b/src/Uno.Templates/Uno.Templates.csproj
@@ -18,7 +18,7 @@
 
 	<PropertyGroup>
 		<UnoExtensionsVersion Condition="'$(UnoExtensionsVersion)' == ''">3.0.0-dev.2335</UnoExtensionsVersion>
-		<UnoVersion Condition="'$(UnoVersion)' == ''">5.0.0-dev.3568</UnoVersion>
+		<UnoVersion Condition="'$(UnoVersion)' == ''">5.0.0-dev.3597</UnoVersion>
 		<UnoExtensionsLoggingVersion Condition="'$(UnoExtensionsLoggingVersion)' == ''">1.6.0</UnoExtensionsLoggingVersion>
 		<SkiaSharpVersion Condition="'$(SkiaSharpVersion)' == ''">2.88.6</SkiaSharpVersion>
 		<UnoCoreExtensionsLoggingVersion Condition="'$(UnoCoreExtensionsLoggingVersion)' == ''">4.0.1</UnoCoreExtensionsLoggingVersion>
@@ -30,7 +30,7 @@
 		<UnoUniversalImageLoaderVersion Condition="'$(UnoUniversalImageLoaderVersion)' == ''">1.9.36</UnoUniversalImageLoaderVersion>
 		<UnoWasmBootstrapVersionNet7 Condition="'$(UnoWasmBootstrapVersionNet7)' == ''">7.0.31</UnoWasmBootstrapVersionNet7>
 		<UnoWasmBootstrapVersionNet8 Condition="'$(UnoWasmBootstrapVersionNet8)' == ''">8.0.0-dev.306</UnoWasmBootstrapVersionNet8>
-		<UnoMarkupVersion Condition="'$(UnoMarkupVersion)' == ''">5.0.0-dev.501</UnoMarkupVersion>
+		<UnoMarkupVersion Condition="'$(UnoMarkupVersion)' == ''">5.0.0-dev.506</UnoMarkupVersion>
 		<UnoUITestHelpersVersion Condition="'$(UnoUITestHelpersVersion)' == ''">1.1.0-dev.70</UnoUITestHelpersVersion>
 	</PropertyGroup>
 	<PropertyGroup>

--- a/src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj
+++ b/src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj
@@ -270,7 +270,7 @@
         <Compile Update="**\*.xaml.cs">
           <DependentUpon>%(Filename)</DependentUpon>
         </Compile>
-        <PRIResource Include="**\*.resw" />
+        <PRIResource Include="**\*.resw" Exclude="bin\**\*.resw;obj\**\*.resw" />
       </ItemGroup>
     </Otherwise>
   </Choose>

--- a/src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj
+++ b/src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj
@@ -265,12 +265,12 @@
         <!--#endif-->
 
         <!-- Include all images by default - matches the __WindowsAppSdkDefaultImageIncludes property in the WindowsAppSDK -->
-        <Content Include="Assets\**;**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif" Exclude="bin\**;obj\**;**\*.svg" />
-        <Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+        <Content Include="Assets\**;**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);**\*.svg" />
+        <Page Include="**\*.xaml" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
         <Compile Update="**\*.xaml.cs">
           <DependentUpon>%(Filename)</DependentUpon>
         </Compile>
-        <PRIResource Include="**\*.resw" Exclude="bin\**\*.resw;obj\**\*.resw" />
+        <PRIResource Include="**\*.resw" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
       </ItemGroup>
     </Otherwise>
   </Choose>

--- a/src/Uno.Templates/reinstall.ps1
+++ b/src/Uno.Templates/reinstall.ps1
@@ -6,7 +6,7 @@ param(
     [string]$ExtensionsVersion = "3.0.0-dev.2335",
 
     # Version of published Uno.WinUI packages
-    [string]$UnoVersion = "5.0.0-dev.3568"
+    [string]$UnoVersion = "5.0.0-dev.3597"
 )
 
 function RemoveNuGetPackage {

